docs/sources/developers/plugins/add-authentication-for-data-source-plugins.md

```diff
@@ -0,0 +1,158 @@
++++
+title = "Add authentication for data source plugins"
+type = "docs"
+aliases = ["/docs/grafana/latest/plugins/developing/auth-for-datasources/",  "/docs/grafana/latest/developers/plugins/authentication/"]
++++
+
+# Add authentication for data source plugins
+
+This page explains how to authenticate against an third-party API from a data source plugin using the Grafana data source proxy.
```

Comment:
```suggestion
This page explains how to authenticate against a third-party API from a data source plugin using the Grafana data source proxy.
```

Reply:
Unclear antecedent. Which item is using the Grafana data source proxy, authenticate, the third-party API, or the plugin?

Reply:
> authenticate using the Grafana data source proxy

Not sure what a better option would be. Would a comma help before "using"?